build-config/make/grub.make

```diff
@@ -16,21 +17,28 @@ GRUB_BUILD_DIR		= $(MBUILDDIR)/grub
 GRUB_DIR		= $(GRUB_BUILD_DIR)/grub-$(GRUB_VERSION)
 GRUB_I386_DIR		= $(GRUB_BUILD_DIR)/grub-i386-pc
 GRUB_UEFI_DIR		= $(GRUB_BUILD_DIR)/grub-x86_64-efi
-GRUB_I386_COREBOOT_DIR		= $(GRUB_BUILD_DIR)/grub-i386-coreboot
+GRUB_I386_COREBOOT_DIR	= $(GRUB_BUILD_DIR)/grub-i386-coreboot
```

Comment:
I revised the indentation by removing one tab space for consistent with the context.